superset-frontend/src/components/ErrorMessage/ParameterErrorMessage.tsx

```diff
@@ -32,17 +33,76 @@ interface ParameterErrorExtra {
   }[];
 }
 
-const triggerMessage = t('This may be triggered by:');
+const maxDistanceForSuggestion = 2;
+const findMatches = (
+  undefined_parameters: string[],
```

Comment:
can we make these camelCase?

Reply:
Oh, yeah!

I added these to the backend first as snake case, when I added to the frontend I forgot to switch to camel case. :-P